tf/edgeml/graph/bonsai.py

```diff
@@ -3,13 +3,13 @@
 
 import tensorflow as tf
 import numpy as np
+import warnings
 
 
 class Bonsai:
-
     def __init__(self, numClasses, dataDimension, projectionDimension,
                  treeDepth, sigma,
-                 W=None, T=None, V=None, Z=None):
+                 isRegression=None, W=None, T=None, V=None, Z=None):
```

Comment:
`isRegression=None` ?

Shouldn't it be either `True` or `False`

Reply:
The value that it gets is always either 'False' or 'True'. That is handled in helpermethods.py. But in case this might lead to confusion, I'll write it as 'False'.

Reply:
Yes please do.

Reply:
Thanks a lot for noticing it @metastableB. It should be False